spec/index.bs

```diff
@@ -4917,12 +4917,29 @@ GPURenderPassEncoder includes GPUProgrammablePassEncoder;
 GPURenderPassEncoder includes GPURenderEncoderBase;
 </script>
 
-  * {{GPURenderEncoderBase/setIndexBuffer()}}/{{GPURenderEncoderBase/setVertexBuffer()}}:
-      * If `size` is zero, the remaining size (after `offset`) of the {{GPUBuffer}} is used.
-
   * In indirect draw calls, the base instance field (inside the indirect
     buffer data) must be set to zero.
 
+{{GPURenderEncoderBase}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPURenderEncoderBase">
```

Comment:
`GPUProgrammablePassEncoder` should also have `[[pipeline]]`, `[[bindgroups]]` (which contain the dynamic offsets too) so we can validate against them.

It might be simpler to do the validation for `GPUComputePassEncoder` first, then `GPURenderPassEncoder` because the latter has more state.

Reply:
Added the internal slot, but am not using it for anything at the moment. I've got a WIP follow-up PR that focuses on the `setBindGroup()` calls, so I'd like to flesh this stuff out in that PR.